traffic_ops/testing/api/v4/cdns_test.go

```diff
@@ -348,6 +373,46 @@ func GetTestCDNs(t *testing.T) {
 	}
 }
 
+func GetTestCDNsbyDomainName(t *testing.T) {
+	if len(testData.CDNs) < 1 {
```

Comment:
line 382 dereferences `testData.CDNs[1]`, so this needs to check that the length is at least 2 to prevent segfaulting (or it could instead use the `0` element on line 382)

Reply:
updated the code to get the first value of CDN.